bashunit

```diff
@@ -17,17 +17,20 @@ source "$BASHUNIT_ROOT_DIR/src/helpers.sh"
 source "$BASHUNIT_ROOT_DIR/src/upgrade.sh"
 source "$BASHUNIT_ROOT_DIR/src/assertions.sh"
 source "$BASHUNIT_ROOT_DIR/src/runner.sh"
+source "$BASHUNIT_ROOT_DIR/src/main.sh"
 
-###############
-#### MAIN #####
-###############
-
+_ASSERT_FN=$_DEFAULT_ASSERT_FN
```

Comment:
Todo: rollback to empty str directly here. This is not a value from the .env but an option passed via the command line only and it should be always empty by default.

Reply:
It's not, but it could be and use a specific .env for CI

Reply:
It's not, but it could be and use a specific .env for CI

Reply:
This is something that you use with `./bashunit -a|--assert`. I don't see how it "could be used for CI" from the `.env`? Or am I missing something? Could you elaborate a bit more on what's the value on having this here, @khru?

Reply:
After talking with @khru and @antonio-gg-dev we agreed on this, so I will revert it 👍🏼